tests/kuttl/tests/openstackbaremetalset_scale/09-assert.yaml

```diff
@@ -32,21 +32,3 @@ status:
   provisioningStatus:
     readyCount: 1
     state: Error
----
-# We don't know which BMH was chosen as compute-0 originally, so we use a script to
-# check for the "BaremetalHostNotFound" error (as the actual error message will vary
-# based on which BMH was actually deleted)
-apiVersion: kuttl.dev/v1beta1
-kind: TestAssert
-namespaced: true
-timeout: 600
-commands:
-  - script: |
-      set -x
-      sleep 5
-      osbmsstate=$(oc get osbms -n openstack compute -o json | jq -r '.status.provisioningStatus.reason')
-      if grep -q "Please check BaremetalHost resources and re-add" <<< "$osbmsstate"; then
-        exit 0
-      else
-        exit 1
-      fi
```

Comment:
This was unnecessary.  Just checking the OSBMS `status.provisioningStatus.readyCount` and `status.provisioningStatus.state` is enough to detect the problem we are expecting.